setup/updating.md

```diff
@@ -28,6 +28,13 @@ helm repo add coder https://helm.coder.com
 Please also ensure that your Kubernetes config is pointing to the cluster on
 which you've deployed Coder.
 
+## Recommendations
+
+- Before beginning, we **strongly recommend** taking snapshots
+of your environment in case there are issues with the update process.
+- We recommend updating no more than version at a time (e.g., we recommend
```

Comment:
I'm curious how much of a jump we want to support, whether we're talking about 1.15 -> 1.16 or also including minor revisions?